theme/banner/_base.scss

```diff
@@ -2,7 +2,10 @@
  Banner
  */
 
+// Needed for backwards compatibility with WET 4.0.27 and below's HTML markup.
+// Remove object references from all of this theme's logo selectors in WET 4.1+.
 .wb-sppe {
+	img,
```

Comment:
Where and when the CSS class ```.wb-sppe``` is used? I am not able to find any reference or description of it.

Reply:
It was first introduced it in #5293 to prevent the splash page's logo from shrinking when printing in IE (including IE11). Then the HTML markup that referenced that class was apparently inadvertently removed in #6113. Since then, the SCSS selector for it has been present in the WET theme, but wasn't tied to any HTML markup to make it take effect.

As a part of this PR I needed a way of targeting the splash page's logo image. I was going to introduce my own class for that purpose, but then stumbled upon the ``wb-sppe`` class and looked into its history. Since it was trying to achieve the same purpose as I was and was accidentally removed, I figured the best way forward would be to restore it.